app/models/piece.rb

```diff
@@ -1,8 +1,21 @@
 class Piece < ActiveRecord::Base
+  belongs_to :user
   belongs_to :game
 
   def is_obstructed?
     return false
   end
 
+#Capture Pieces Methods - Validates if the other piece is valid to capture, moves current piece to new location
+
+  def move_to!(new_x, new_y)
+    capture_piece = Piece.find_by(x_cord: new_x, y_cord: new_y)
```

Comment:
Few thoughts here:
* lets think about a better name for capture_piece. It sounds like a verb and is a bit confusing imho - maybe piece_to_capture
* what would you expect from Piece.find_by(x, y)? In a year from now I would expect thousands of pieces as a result.. :-) Getting my point?